topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperatorUtil.java

```diff
@@ -158,6 +172,16 @@ public static <K, V> PartitionedByError<K, V> partitionedByError(Stream<Pair<K,
         return new PartitionedByError<>(collect.get(true), collect.get(false));
     }
 
+    /**
+     * Get KafkaTopic resource version.
+     *
+     * @param kafkaTopic Kafka topic.
+     * @return Resource version.
+     */
+    public static String resourceVersion(KafkaTopic kafkaTopic) {
+        return kafkaTopic == null || kafkaTopic.getMetadata() == null ? "null" : kafkaTopic.getMetadata().getResourceVersion();
```

Comment:
I guess the call to `getResourceVersion()` could return `null`. Given the effort gone to to return `"null"`, I suspect that's something you want to avoid.